src/targetsomstandalone.py

```diff
@@ -17,7 +17,7 @@ def entry_point(argv):
     except Exception, e:
         os.write(2, "ERROR: %s thrown during execution.\n" % e)
         return 1
-    return 0
+    return 1
```

Comment:
Why return 1 here? I don't think that a SOM application has to do `system exit: 0` to exit correctly.
Normally, when `System initialize: args` returns, I would expect it to get here without exception, no?